numpy/testing/utils.py

```diff
@@ -4,16 +4,15 @@
 
 """
 import warnings
+from ._private.utils import *
```

Comment:
This is a temporary compatibility file, best leave it alone as it will be removed at some point.